### ISSUE TITLE
PDF Height Overflow

### ISSUE THREAD

**Issue by tresf** [qzind/tray#154]
If a PDF page height of 200 inches is supplied, QZ Tray crashes with an `OutOfMemoryError`, possibly an overflow calculation within the software.

Unlike other `OutOfMemory` errors, this error is NOT due to the JVM running out of memory. The JVM holds steady at `~256MB` used, it never actually runs out.

Although 200 inches may seem high, for continuous feed paper 72 inches is quite common of a size.

QZ Tray should catch the overflow exception or be patched to handle it properly.

**Comment by bberenz** [comment]
This looks like it is caused by the JVM running out of memory:
```
[TRACE] 2017-03-03 17:20:09,262 @ qz.printer.action.PrintPixel:83
	Drawable area: 0.0,0.0:8.5,200.0
...
[DEBUG] 2017-03-03 17:20:12,074 @ qz.printer.BookBundle:38
	Paper area: 0,0:612,14400
[ERROR] 2017-03-03 17:20:12,634 @ qz.ws.PrintSocketClient:141
	Connection error
Caused by: java.lang.OutOfMemoryError: Java heap space
```

Just tried bumping up the max memory available to qz - maxes out around 4.5GB to successfully print an 8.5x200in page. (And around 3.5GB for a 8.5x72in)

**Comment by tresf** [comment]
Confirmed, I cannot reproduce the original description where `~256MB` utilized, on 64-bit JVM mine consumes as much memory as needed and succeeds.  Closing, marking duplicate of #27, 72, #139.
